### PR TITLE
Update docs/results.md

### PR DESCRIPTION
**Pull request by ushuz**
- ~~Add an example of setting result TTL with `enqueue()`~~
- Typo
- Encourage the use of `enqueue()`

### CODE REVIEW

File: docs/results.md [hunk 1]
```diff
@@ -91,11 +95,11 @@ job timed out.
 
 If a job requires more (or less) time to complete, the default timeout period
 can be loosened (or tightened), by specifying it as a keyword argument to the
-`Queue.enqueue()` call, like so:
+`enqueue()` call, like so:
 
 {% highlight python %}
 q = Queue()
-q.enqueue(func=mytask, args=(foo,), kwargs={'bar': qux}, timeout=600)  # 10 mins
```
Comment (ushuz) [inline docs/results.md]: `enqueue()` doesn't have `func` keyword argument

File: docs/results.md [hunk 2]
```diff
@@ -108,7 +112,7 @@ high = Queue('high', default_timeout=8)  # 8 secs
 low = Queue('low', default_timeout=600)  # 10 mins
 
 # Individual jobs can still override these defaults
-low.enqueue_call(really_really_slow, timeout=3600)  # 1 hr
```
Comment (ushuz) [inline docs/results.md]: I think here we should use `enqueue()` just like the above snippet as they are in the same section. Besides, `enqueue_call()` usually looks like `enqueue_call(func= really_really_slow)`.

Comment (selwin) [reply docs/results.md]: Yes, agreed. Could you also replace references to `enqueue_call` with `enqueue`? We really want to encourage people to use `enqueue`

Comment (ushuz) [reply docs/results.md]: @selwin I thought the same. I pushed a new commit replacing `enqueue_call()` with `enqueue()`. PTAL.

**Comment by selwin** [comment]
Thanks!

**Pull request merged**